package.json

```diff
@@ -60,10 +61,10 @@
 		"jest": "^29.6.2",
 		"jest-environment-jsdom": "^29.6.2",
 		"prettier": "^3.0.1",
-		"react": "18.1.0",
-		"react-dom": "18.1.0",
+		"react": "18.1.0 || 19.0.0",
```

Comment:
In the DS, we only updated the `peerDependencies`. I didn't even realise you could do ranges here, but 19 has some annoyances with snapshots updating and other perf issues, so we're still planning to dev against 18 for the near future but also test in 19